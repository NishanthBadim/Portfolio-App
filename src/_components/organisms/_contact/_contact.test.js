import React from "react";
import { create } from "react-test-renderer";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Contact from "./_contact";

jest.mock('./../../molecules/_contactForm/_contactForm', () => ({
  __esModule: true,
  default: () => <div>I am mocked</div>
}));

describe("Contact component testing", () => {


  test("renders", () => {
    const { container } = render(<Contact />);
    expect(container.textContent).toMatch("I am mocked");
  });

  test("Matches the snapshot", () => {
    const input = create(<Contact />);
    expect(input.toJSON()).toMatchSnapshot();
  });
});