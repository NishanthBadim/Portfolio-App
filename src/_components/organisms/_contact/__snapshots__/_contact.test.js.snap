// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component testing Matches the snapshot 1`] = `
<div
  className="makeStyles-root-5"
>
  <div
    className="mdl-grid makeStyles-contactGrid-6"
  >
    <div
      className="mdl-cell mdl-cell--8-col"
    >
      <h3
        className="makeStyles-h3-7"
      >
        Share your details and get in touch with me.
      </h3>
      <hr
        className="makeStyles-hr-8"
      />
      <div
        className="contact-list"
      >
        <div>
          I am mocked
        </div>
      </div>
    </div>
  </div>
</div>
`;
